addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -9383,6 +9385,96 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    QUnit.test('onchange affecting inline unopened list view', function (assert) {
+        // when we got onchange result for fields of record that were not
+        // already available because they were in a inline view not already
+        // opened, in a given configuration the change were applied ignoring
+        // existing data, thus a line of a one2many field inside a one2many
+        // field could be duplicated unexplectedly
+        assert.expect(5);
+
+        var numUserOnchange = 0;
+
+        this.data.user.onchanges = {
+            partner_ids: function (obj) {
+                if (numUserOnchange === 0) {
+                    // simulate proper server onchange after save of modal with new record
+                    var addedTurtle = obj.partner_ids[1][2].turtles[0];
```

Comment:
i got confused by the variable name, so i suggest to rename it s.t. it reflects that this is actually a command, or not to use a variable as it is used only once anyway